nums/numpy/api/creation.py

```diff
@@ -1116,3 +1116,77 @@ def zeros_like(prototype, dtype=None, order="K", shape=None):
     if order is not None and order != "K":
         raise NotImplementedError("Only order='K' is supported.")
     return zeros(shape, dtype)
+
+
+def asarray(a, dtype=None, order=None, *, like=None):
+    """Convert the input to an array.
+
```

Comment:
Add the lines:

```
This docstring was copied from numpy.asarray.

Some inconsistencies with the NumS version may exist.
```

This is just to be consistent with the other documentation.

Reply:
I manually checked the docstring. There shouldn't be an inconsistencies

Reply:
This is just a warning on all of the NumPy API docstrings

Reply:
I hear you. I see how adding the lines improves consistency. That being said, I don't think we should warn users about inconsistencies if we know there are none.

Reply:
Sure, if @elibol approves, then I'm fine with not including this and resolving the followup with the doctest warning.

Other than that looks consistent with the documentation

Reply:
sounds good to me.